ui/gulpfile.esm.js

```diff
@@ -28,6 +28,15 @@ task(
   shell.task('yarn react-app-rewired build --library')
 )
 
+task(
```

Comment:
Is it better to use gulp style file process? e.g. input a file, transform, and finally output.

Reply:
Sounds good, let me try it.